### ISSUE TITLE
Responsive Design

### ISSUE THREAD

**Issue by i-am-SnehaChauhan** [alfg/opendrinks#1378]
**Not Responsive**
The current implementation of the project/application lacks responsiveness, causing usability issues on various devices with different screen sizes.

**Steps to Reproduce:**
Open the application on a mobile device with a small screen.
Navigate to https://opendrinks.io/.
Observe that the name is not visible completely.

**Expected behavior**
The user interface should gracefully adapt to different screen sizes, ensuring a seamless experience on devices ranging from desktops to mobile phones.

**Screenshots**
![image](https://github.com/alfg/opendrinks/assets/94298791/158982ec-d58f-4a6a-80c9-3a643fc6e621)

**Assignment Request:**
I request you to assign this issue to me under #hacktoberfest.

**Comment by alfg** [comment]
Hi @i-am-SnehaChauhan, which device and resolution is this? We have responsive design, but might need to accommodate more resolutions.

**Comment by i-am-SnehaChauhan** [comment]
Yeah, I just checked it again and found that the overall design is responsive. However, the current advertisement content lacks responsiveness, which may result in potential usability issues on various devices and screen sizes. This issue aims to address the need for making the advertisement content responsive for an enhanced user experience.
![image](https://github.com/alfg/opendrinks/assets/94298791/d541fed7-e469-46e8-acfd-064e659d38fa)

**Comment by alfg** [comment]
That's strange. I thought the adsense ads would resize based on the browser/resolution. Which browser are you using here?

**Comment by i-am-SnehaChauhan** [comment]
I am using Chrome.
